### ISSUE TITLE
Criar primeiro tutorial e setup com Github Pages

### ISSUE THREAD

**Issue by wandersoncferreira** [wandersoncferreira/clojure-brasil#9]
@ericdallo comentou sobre utilizar Material Markdown para publicar os tutoriais postados aqui como uma página web via Github Pages.

Eric, se você puder descrever um pouco o setup necessário aqui na issue de como você está pensando em fazer isso dado a experiencia nos repos do lsp-mode e clojure-lsp.

**Comment by ericdallo** [comment]
Claro, a idéia era usar o [mkdocs-material](https://github.com/squidfunk/mkdocs-material), um gerador de site estatico a partir de markdown somente, alguns sites ja usam como: [Calva](https://calva.io/), [clojure-lsp](https://clojure-lsp.github.io/clojure-lsp/), [Emacs-LSP](https://emacs-lsp.github.io/lsp-mode/) e outros.

Basicamente precisariamos de um `mkdocs.yaml` configurando tema e outras opcoes do site e uma pasta `docs` contendo arquivos/pastas com os markdows. Poderiamos configurar um CI como [este](https://github.com/emacs-lsp/lsp-dart/blob/master/.github/workflows/docs.yml) onde pusharia para uma branch e hospedariamos de graça via Github Pages.

**Comment by edipofederle** [comment]
Olá Pessoal, qual o status disso?

**Comment by ericdallo** [comment]
Poderia ser neste aqui mesmo, não vejo nenhum problema

**Comment by edipofederle** [comment]
Acabei puxando algumas coisas aqui, mas me dei conta que já temos o https://github.com/clj-br/clj-br.github.io (http://clojurebrasil.com.br/), esse é feito com Clojure (basicamente com templates hiccup2). 

Acho que seria interessante termos um único lugar para tudo ?  talvez fazer tudo via mkdocs?

O que acham?

**Comment by ericdallo** [comment]
Concordo, acho o mkdocs muito fácil de manter onde qualquer um pode facilmente criar/editar algo

**Comment by edipofederle** [comment]
@pbalduino acho que voce que criou/mantem o clj-br/clj-br o que acha sobre termos um único lugar e via mkdocs, como discutido acima?
